dev/com.ibm.ws.microprofile.faulttolerance/test/src/com/ibm/ws/microprofile/faulttolerance/test/AsyncTest.java

```diff
@@ -49,7 +49,7 @@
     @Test
     public void testAsync() throws InterruptedException, ExecutionException, TimeoutException {
         ExecutorBuilder<String, String> builder = FaultToleranceProvider.newExecutionBuilder();
-        Executor<Future<String>> executor = builder.buildAsync();
+        Executor<Future<String>> executor = builder.buildAsync(Future.class);
```

Comment:
Add some equivalent tests for CompletionStage

Reply:
in the 2.0 bundle

Reply:
We do have a test for `CompletionStage` in `AsyncTest` in the 2.0 bundle.